pkgs/applications/audio/mixxx/default.nix

```diff
@@ -2,8 +2,11 @@
 , libopus, libshout, libsndfile, libusb1, libvorbis, pkgconfig
 , portaudio, portmidi, protobuf, qt4, rubberband, scons, sqlite
 , taglib, vampSDK
+, aacSupport ? false, faad2 ? null, mp4v2 ? null
```

Comment:
I would actually prefer this not to be optional unless there is a good reason to.